account_mass_reconcile/models/mass_reconcile.py

```diff
@@ -301,7 +301,7 @@ def _get_date(reconcile):
         if run_all:
             reconciles.run_reconcile()
             return True
-        reconciles.sorted(key=_get_date)
-        older = reconciles[0]
+        ordered_reconciles = reconciles.sorted(key=_get_date)
+        older = ordered_reconciles[0]
```

Comment:
You should put `older = order_reconciles[:1]` for avoiding error in case there's no reconciles.